daemons/compute/lib-cpp/client.cc

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright 2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <memory>
+#include <iostream>
+
+#include <grpcpp/grpcpp.h>
+
+#include "client.h"
+#include "datamovement.grpc.pb.h"
+
+class DataMoverClientInternal {
```

Comment:
I suggest that you place all of these classes in:
```
namespace nnfdmc {
}
```

Our whichever namespace you choose.  I used nnfdmc for nnf data mover client.